string-manipulation-2/src/first-chars.js

```diff
@@ -0,0 +1,12 @@
+/* exported firstChars */
+function firstChars(length, string) {
+  let newWord = '';
+  if (length > string.length) {
+    newWord = string;
+  } else if (length < string.length) {
+    for (let i = 0; i < length; i++) {
+      newWord += string[i];
+    }
+  }
+  return newWord;
+}
```

Comment:
We can actually accomplish this without a loop! We can use `slice`. Check out the snippet below
```
function firstChars(length, string) {
  return string.slice(0, length);
}
```

Reply:
Thanks Shawn! I didn't think about using slice() !